mobile/src/main/java/ch/epfl/sdp/mobile/ui/i18n/English.kt

```diff
@@ -191,3 +192,16 @@ object English : LocalizedStrings {
   override val tournamentsFilterOnlyAdministrating = "Administrating".uppercase()
   override val tournamentsFilterBackContentDescription = "Back"
 }
+
+/** Converts a [Duration] to an English string. */
+private fun Duration.toEnglishString(): String {
+  if (this.toString().contains("d")) {
+    return "${this.toLong(DurationUnit.DAYS)} days"
+  } else if (this.toString().contains("h")) {
+    return "${this.toLong(DurationUnit.HOURS)} hours"
+  } else if (this.toString().contains("m")) {
+    return "${this.toLong(DurationUnit.MINUTES)} minutes"
+  } else {
+    return "${this.toLong(DurationUnit.SECONDS)} seconds"
+  }
+}
```

Comment:
You may not want to match on the string representation of the `Duration`. Shouldn't you see if it's above a certain threshold, and then use the `inWholeDays` and co. methods to retrieve the duration ?

Also, what would you say of performing the rounding of durations here directly ?